### ISSUE TITLE
Reciprocal FIJMIX

### ISSUE THREAD

**Issue by ianhbell** [usnistgov/REFPROP-issues#152]
When a mixture is loaded backwards to the order in the ``HMX.BNC`` file, the reciprocal values for betaT and betaV are incorrectly returned from REFPROPdll.

Example:

``` python
from ctREFPROP.ctREFPROP import REFPROPFunctionLibrary
root = 'D:/Program Files (x86)/REFPROP'
RP = REFPROPFunctionLibrary(root)
MOLAR_BASE_SI = RP.GETENUMdll(0,'MOLAR BASE SI').iEnum
RP.SETPATHdll(root)

r = RP.REFPROPdll('PROPANE;H2S','','FIJMIX',MOLAR_BASE_SI,0,1,1,2,[z0,1-z0]+[0.0]*18)
print(r.Output[0:5], r.ierr, r.herr)
```
yields
```
array('d', [1.007482008719, 0.905829247, 1.067450922575, 1.010593999, 0.0]) -852 [ALLPROPS warning -852] Additional properties are included in the output array, the input string and output array do not line up.
```
but should yield the values 
```
0.992573556, 0.905829247, 0.936811219, 1.010593999
```
because this is the relevant chunk from the HMX.BNC:
```
?Hydrogen sulfide/Propane                                  [H2S/C3]
?Kunz and Wagner (2012)
?#The values of betaT and betaV are reciprocals of the published values, but deviations are expected to be less than 1 ppb.
  c6c03020/70c6aac0
    KW0      1.007482008719 0.905829247    1.067450922575 1.010593999    0.             0.             0. 0. 0. 0. 0. 0.
```

I checked the same calculation with CoolProp, and the NBP temperatures are the same to 8 digits, so internally, betaT and betaV must be being handled correctly, just not being returned correctly.

**Comment by ianhbell** [comment]
Ping.  Hit this one again.

**Comment by ianhbell** [comment]
Also returned incorrectly from ``GETKTVdll``

**Comment by EricLemmon** [comment]
This one is now fixed, I will put the details in #296

**Comment by ianhbell** [comment]
Fixed.
``` python
from ctREFPROP.ctREFPROP import REFPROPFunctionLibrary
import os
root = 'D:/Program Files (x86)/REFPROPalpha'
RP = REFPROPFunctionLibrary(root)
print(RP.RPVersion())
RP.SETPATHdll(root)
RP.SETFLUIDSdll('ARGON')
print(RP.REFPROPdll('PROPANE*CF3I', '', 'FIJMIX', RP.MASS_BASE_SI, 0, 0, 1, 2, [0.5, 0.5]).Output[0:6])
print(RP.GETKTVdll(1, 2))
RP.SETFLUIDSdll('ARGON')
print(RP.REFPROPdll('CF3I*PROPANE', '', 'FIJMIX', RP.MASS_BASE_SI, 0, 0, 1, 2, [0.5, 0.5]).Output[0:6])
print(RP.GETKTVdll(1, 2))
```
yields
```
10.0078
array('d', [1.0055102, 0.98807, 1.0, 1.0, 0.0, 0.0])
GETKTVdlloutput(hmodij='XR0', fij=array('d', [1.0055102, 0.98807, 1.0, 1.0, 0.0, 0.0]), hFmix='HMX.BNC', hfij='BetaT   GammaT  BetaV   GammaV not usednot used', hbinp='Propane/R13I1\r\nI.H. Bell and E.W. Lemmon, JCED, 2016. DOI:10.1021/acs.jced.6b00257; MAPD: 0.26% from 24 bubble-point pressures', hmxrul='XR0  Reducing functions only')
array('d', [0.9945199959184899, 0.98807, 1.0, 1.0, 0.0, 0.0])
GETKTVdlloutput(hmodij='XR0', fij=array('d', [0.9945199959184899, 0.98807, 1.0, 1.0, 0.0, 0.0]), hFmix='HMX.BNC', hfij='BetaT   GammaT  BetaV   GammaV not usednot used', hbinp='Propane/R13I1\r\nI.H. Bell and E.W. Lemmon, JCED, 2016. DOI:10.1021/acs.jced.6b00257; MAPD: 0.26% from 24 bubble-point pressures', hmxrul='XR0  Reducing functions only')
```
